File: c#/vs/Common7/IDE/ProjectTemplatesCache/CSharp/WCF/2052/SequentialWorkflowServiceLibrary/IWorkflow1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ServiceModel;

namespace $safeprojectname$
{
	// 注意: 使用“重构”菜单上的“重命名”命令，可以同时更改代码和配置文件中的接口名“IWorkflow1”。
	[ServiceContract]
	public interface IWorkflow1
	{

		[OperationContract]
		string GetData(int value);

		// TODO: 在此添加您的服务操作
	}
}

File: c#/vs/Common7/IDE/ProjectTemplatesCache/VisualBasic/WCF/2052/SequentialWorkflowServiceLibrary/IWorkflow1.vb
' 注意: 使用上下文菜单上的“重命名”命令可以同时更改代码和配置文件中的接口名“IWorkflow1”。
<ServiceContract()> _
Public Interface IWorkflow1

    <OperationContract()> _
    Function GetData(ByVal value As Integer) As String

    ' TODO: 在此添加您的服务操作

End Interface
